src/cdk/drag-drop/directives/drag.ts

```diff
@@ -356,6 +379,33 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
     return element;
   }
 
+  /**
+   *  Gets the element for the preview clone to be created in,
+   *  based on the `previewContainer` value.
+   */
+  private _getPreviewContainer() {
+    const cloneTarget = this.previewContainer;
+
+    if (!cloneTarget) {
+      return null;
+    }
+
+    if (typeof cloneTarget === 'string' && cloneTarget === 'global') {
+      // Explicitly returning null for this case as logic in `DragRef` controls whether this is
+      // body or fullscreenElement (incl. vendor prefixes).
+      return null;
+    }
+
+    if (typeof cloneTarget === 'string' && cloneTarget === 'parent') {
+      return this.dropContainer
+        ? this.dropContainer.element.nativeElement.parentElement
```

Comment:
I think that this should use `this.element.nativeElement.parentElement` since the drag item won't necessarily be directly inside the drop list.

Reply:
I havent seen any issues when using it outside of a drop list.
It seems that the clone is only created when used with a drop list? I'm still learning the cdk so if i've missed a use case please let me know and i'll check it out

Reply:
You're right that the preview is only created when the item is inside of a list. What I meant above is that we'd want the clone to be next to the item being dragged, because the list itself might have some styles that are inherited by the items.

Reply:
i guess the outcome of this is related to the other conversation, so i'm gonna mark this as resolved so we're not discussing in two places 👍